cloudinit/distros/tests/test_init.py

```diff
@@ -25,14 +36,16 @@ class TestGetPackageMirrorInfo:
         # Empty info gives empty return
         ({}, {}),
         # failsafe values used if present
-        ({'failsafe': {'primary': 'value', 'security': 'other'}},
-         {'primary': 'value', 'security': 'other'}),
+        ({'failsafe': {'primary': 'http://value', 'security': 'http://other'}},
```

Comment:
why do we add the 'http://' strings here now?

Reply:
Just for consistency with the other test data (and it does make it a little clearer which of these strings is the mirror value, but that's not a huge gain).

Reply:
OK.  I'm always worried that changing the test-data obscures a bug or case we're not handling.  I don't think that's the case here since we are parsing URLs.  +1 on the change.